app/models/visit_metrics_entry.rb

```diff
@@ -1,6 +1,6 @@
 class VisitMetricsEntry < ActiveRecord::Base
   validates_presence_of :visit_id, :nomis_id, :requested_at
-  validates_inclusion_of :kind, in: %w{deferred instant}
+  validates_inclusion_of :kind, in: %w{deferred}
```

Comment:
It would also involve some changes to metrics, which don't have any test coverage, and which I'd rather leave entirely alone until we replace them with something working.